packages/d3fc-discontinuous-scale/src/tickFilter.js

```diff
@@ -1,11 +1,19 @@
+import { pairs } from 'd3-array';
+
+function areEqual([a, b]) {
+    if (a.valueOf) {
```

Comment:
Everything in JS has a `valueOf` i.e. [`Object.prototype.valueOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf) other than `null`/`undefined`. However, they'd cause a `TypeError` here so I guess that's not what you're aiming to catch.

How about a `values.map(d => d?.valueOf())`?